---
layout: single
author_profile: false
share: true
title: "The effect of preprocessing by sequence-independent, single-primer amplification (SISPA) on metagenomic detection of viruses."
authors: "Karlsson OE, Bel√°k S, Granberg F."
excerpt: ""
---

*{{ page.authors }}*

### Abstract

Compared to routine diagnostics, screening for pathogens in outbreak
situations, with or without intentional release, poses demands on the detection
technology to not only indicate the presence of already known causative agents
but also novel and unexpected pathogens. The metagenomic approach to detecting
viral pathogens, using unbiased high-throughput sequencing (HTS), is a
well-established methodology with a broad detection range and wide
applicability on different sample matrices. To prepare a sample for HTS, the
common presequencing steps include homogenization, enrichment, separation
(eg, magnetic separation), and amplification. In this initial study, we
explored the benefits and drawbacks of preprocessing by sequence-independent,
single-primer amplification (SISPA) of nucleic acids by applying the
methodology to artificial samples. More specifically, a synthetic metagenome
was divided into 2 samples, 1 unamplified and 1 diluted, and amplified by
SISPA. Subsequently, both samples were sequenced using the Ion Torrent Personal
Genome Machine (PGM), and the resulting datasets were analyzed by using
bioinformatics, short read mapping, de novo assembly, BLAST-based taxonomic
classification, and visualization. The results indicate that even though SISPA
introduces a strong amplification bias, which makes it unsuitable for
whole-genome sequencing, it is still useful for detecting and identifying
viruses.

### Read more

[Full article link](http://online.liebertpub.com/doi/full/10.1089/bsp.2013.0008)
