---
layout: single
author_profile: false
share: true
title: "Simulating Illumina Metagenomic Data with InSilicoSeq"
authors: "Hadrien Gourlé, Oskar Karlsson-Lindsjö, Juliette Hayer, Erik Bongcam-Rudloff"
excerpt: ""
---

*{{ page.authors }}*

### Abstract

**Motivation:** The accurate in-silico simulation of metagenomic datasets is of great importance for benchmarking bioinformatics tools as well as for experimental design. Users are dependant on large-scale simulation to not only design experiments and new projects but also for accurate estimation of computational needs within a project. Unfortunately, most current read simulators are either not suited for metagenomics, out of date or relatively poorly documented. In this article, we describe InSilicoSeq, a software package to simulate metagenomic Illumina sequencing data. InsilicoSeq has a simple command-line interface and extensive documentation.  
**Results:** InSilicoSeq is implemented in Python and capable of simulating realistic Illumina (meta) genomic data in a parallel fashion with sensible default parameters.  
**Availability:** Source code and documentation are available under the MIT license at <https://github.com/HadrienG/InSilicoSeq> and <https://insilicoseq.readthedocs.io/>.

### Read more

[Full article link](https://doi.org/10.1093/bioinformatics/bty630)
